srcpkgs/pacman/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'pacman'
+pkgname=pacman
+version=5.2.1
+revision=1
+wrksrc="pacman-${version}"
+make_dirs="/var/lib/pacman 0755 root root
+ /usr/var/cache/pacman/pkg 0755 root root
+ /usr/share/libalpm/hooks 0755 root root
+ /usr/share/pacman/keyrings 0755 root root"
+conf_files="/etc/pacman.conf"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc
+ --localstatedir=/var"
+hostmakedepends="autoconf automake libtool gettext-devel pkg-config"
+makedepends="libarchive-devel gpgme-devel libcurl-devel"
+depends="gnupg2"
+short_desc="Arch Linux's library-based package manager"
```

Comment:
I don't think the description is good.

Reply:
I updated the description to use Fedora's, is that better?

Reply:
No, I should have been clearer about that, I don't think "Arch" or "Arch Linux" is really necessary or useful here, I would prefer something like pacmans own description:
 
> Simple library-based package manager

https://repology.org/project/pacman/information#Summaries

Reply:
Ah thanks.